### PR TITLE
docs: add docs about wbtc transfer using erc-2771 and erc-4337

### PR DESCRIPTION
**Pull request by peterslany**
Adding docs for WBTC gas fee payment using ERC-2771 and ERC-4337 standards.

### CODE REVIEW

**Comment by nud3l** [comment]
@sander2 can you please review this first and then I can do a final review?

File: docs/docs/build/examples/wbtc-gas-fee-erc-2771/index.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Gas fee payment in WBTC using ERC-2771
```
Comment (sander2) [inline docs/docs/build/examples/wbtc-gas-fee-erc-2771/index.md:1]:
```suggestion
# Gas fee payment in WBTC using OpenGSN & ERC-2771
```

ERC-2771 is only a small part of opengsn - I would put a little bit more emphasis on the opengsn part, also in the into below

File: docs/docs/build/examples/wbtc-gas-fee-erc-2771/index.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+:::info Example Code
+
+The source code for this demo can be found in [this repository](https://github.com/bob-collective/demo-meta-transactions-transfer/).
+
+:::
+
+![preview](preview.png)
+
+## Local development
```
Comment (sander2) [inline docs/docs/build/examples/wbtc-gas-fee-erc-2771/index.md:13]: I guess local setup can be useful as well, but didn't we want to host the page ourselves as well?

File: docs/docs/build/examples/wbtc-gas-fee-erc-2771/index.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
+# Gas fee payment in WBTC using ERC-2771
+
+In this example, we will show how WBTC can be used for gas fee payments using the [ERC-2771 standard](https://eips.ethereum.org/EIPS/eip-2771) on the BOB testnet. This enables users to transact without the necessity to own ether.
```
Comment (sander2) [inline docs/docs/build/examples/wbtc-gas-fee-erc-2771/index.md:3]: I think we should explain a little bit about openGSN. It doesn't have to be super thorough - we can link to their docs as well. Just the basics of how it works, and how to support it in contracts. (the latter part could be under a separate header though)

File: docs/docs/build/examples/wbtc-gas-fee-erc-4337/index.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+:::info Example Code
+
+The source code for this demo can be found in [this repository](https://github.com/bob-collective/demo-account-abstraction-transfer/).
+
+:::
+
+![preview](preview.png)
+
+## Local development
```
Comment (sander2) [inline docs/docs/build/examples/wbtc-gas-fee-erc-4337/index.md:13]: again, shouldn't we host this as well?

File: docs/docs/build/examples/wbtc-gas-fee-erc-2771/index.md (hunk 2 above)
Comment (nud3l) [reply docs/docs/build/examples/wbtc-gas-fee-erc-2771/index.md:13]: Yes, we need to host these demos.

File: docs/docs/build/examples/wbtc-gas-fee-erc-4337/index.md (hunk 4 above)
Comment (nud3l) [reply docs/docs/build/examples/wbtc-gas-fee-erc-4337/index.md:13]: yes, we need to host this as well

File: docs/docs/build/examples/wbtc-gas-fee-erc-4337/index.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
+# Gas fee payment in WBTC using ERC-4337
+
+In this example, we will show how WBTC can be used for gas fee payment using the [ERC-4337 account abstraction standard](https://eips.ethereum.org/EIPS/eip-4337) on the BOB testnet. This enables users to use smart contract wallets and transact without the need to own ether.
```
[the remark this answers is not in the document]
Comment (nud3l) [reply docs/docs/build/examples/wbtc-gas-fee-erc-4337/index.md:3]: agree with Sander

File: docs/docs/build/examples/wbtc-gas-fee-erc-2771/_category_.yml [hunk 6]
```diff
@@ -0,0 +1,3 @@
+position: 4
```
Comment (nud3l) [inline docs/docs/build/examples/wbtc-gas-fee-erc-2771/_category_.yml:1]: Let's merge both AA and MT into the how to section under a combined title called "Using bridged BTC as Transaction Fees".

The first paragraph should then be a comparison of AA and MT with their trade-offs (can be taken from the knowledge sharing presentation)

File: docs/docs/build/examples/wbtc-gas-fee-erc-2771/index.md (hunk 1 above)
Comment (nud3l) [reply docs/docs/build/examples/wbtc-gas-fee-erc-2771/index.md:1]: Please also use the common names in the heading - Meta Transactions

File: docs/docs/build/examples/wbtc-gas-fee-erc-4337/index.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
+# Gas fee payment in WBTC using ERC-4337
```
Comment (nud3l) [inline docs/docs/build/examples/wbtc-gas-fee-erc-4337/index.md:1]:
```suggestion
# Gas Fee Payment in WBTC using Account Abstraction (ERC-4337)
```

File: docs/docs/build/examples/wbtc-gas-fee-erc-4337/index.md (hunk 5 above)
Comment (nud3l) [inline docs/docs/build/examples/wbtc-gas-fee-erc-4337/index.md:3]:
```suggestion
In this example, we will show how WBTC can be used for gas fee payment using the [ERC-4337 account abstraction standard](https://eips.ethereum.org/EIPS/eip-4337) on the BOB testnet. This enables users to use smart contract wallets and transact without the need to own ETH.
```

**Review by nud3l — CHANGES_REQUESTED** [review]
Main things that need addressing:
1. Move the docs into the how to section to explain how to achieve paying for tx fees in bridged BTC
2. Compare the trade-offs of AA and MT
3. Host both demos on the testnet

**Pull request merged**